XenAdmin/Wizards/PatchingWizard/PatchingWizard_UploadPage.cs

```diff
@@ -49,7 +49,7 @@ public PatchingWizard_UploadPage()
         }
 
         #region Accessors
-        public readonly List<HostUpdateMapping> PatchMappings = new List<HostUpdateMapping>();
+        public List<HostUpdateMapping> PatchMappings = new List<HostUpdateMapping>();

```

Comment:
Following SonarQube's _"Mutable, non-private fields should not be "readonly"_ here:

> Using the readonly keyword on a field means that it can't be changed after initialization. However, when applied to collections or arrays, that's only partly true. readonly enforces that another instance can't be assigned to the field, but it cannot keep the contents from being updated. That means that in practice, the field value really can be changed, and the use of readonly on such a field is misleading, and you're likely to not be getting the behavior you expect.

Reply:
Ok. Maybe make it a get-only property instead? `public List<HostUpdateMapping> PatchMappings {get;} = new List<HostUpdateMapping>();`